OPHD/Common.h

```diff
@@ -24,6 +24,18 @@ enum class Difficulty
 };
 
 
+inline std::map<std::string, Difficulty> difficultyTable
+{
+	{"Beginner", Difficulty::Beginner},
+	{ "Easy", Difficulty::Easy },
+	{ "Medium", Difficulty::Medium },
+	{ "Hard", Difficulty::Hard }
+};
```

Comment:
Whitespace is a bit inconsistent between the first entry and the last 3 entries.

----

Should probably mark this as `constexpr` (or at least `const`). Using `constexpr` could replace the use of `inline`.

Reply:
It seems `constexpr` is not possible here, since the type is considered non-literal. Not sure if that's because of the `std::map` or because of the `std::string`. I doubt it's because of the `enum`.

> OPHD/States/../Common.h:27:45: error: constexpr variable cannot have non-literal type 'const std::map<std::string, Difficulty>' (aka 'const map<basic_string<char>, Difficulty>')
> constexpr std::map<std::string, Difficulty> difficultyTable

I went with `inline const` instead.

Reply:
Yeah, std::string is a non-literal. Not sure about std::map, but you definetly cannot declare any container of std::strings as constexpr from experience.